### PR TITLE
Update overdue list UI

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/3736/update-the-ui-for-the-overdueappointmentrow

### CODE REVIEW

**Comment by msasikanth** [comment]
@vinaysshenoy can you approve these code climate issues, please? There is no avoiding the 5 argument limit here. It's a factory constructor 😅

**Review by Honey14 — CHANGES_REQUESTED** [review]
Overall looks good. Can you check the android tests? And waiting on Vinay to approve the codeclimate issues

File: app/src/main/res/drawable/ic_overdue_call.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="40dp"
```
[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/main/res/drawable/ic_overdue_call.xml:2]: That's a normal call icon. This is a call icon inside a circle shape.

**Review by Honey14 — DISMISSED** [review]
LGTM. @vinaysshenoy Can you approve the codeclimate issues in this PR?

**Comment by vinaysshenoy** [comment]
> @vinaysshenoy can you approve these code climate issues, please? There is no avoiding the 5 argument limit here. It's a factory constructor sweat_smile
> 
> One way I thought of avoiding this issue was to use Dagger inject to instead of the clock and date formatter. But didn't want to make the adapter part of Dagger.

Cool. For what it's worth, one of the parameters is a feature flag, so this will eventually get removed anyway,

**Review by Honey14 — APPROVED** [review]

**Pull request merged**